test/functional/core/startup_spec.lua

```diff
@@ -400,9 +400,9 @@ describe('startup', function()
         read_file('Xtest_startup_ttyout')
       )
     end)
-    if is_os('win') then
-      assert_log('stream write failed. RPC canceled; closing channel', testlog)
-    end
+    -- if is_os('win') then
+    --   assert_log('stream write failed. RPC canceled; closing channel', testlog)
+    -- end
```

Comment:
Tests pass with this commented and fail with it uncommented. I guess whatever issue caused this message in the log is fixed?